sled-agent/src/params.rs

```diff
@@ -249,16 +249,30 @@ impl From<DatasetEnsureBody> for sled_agent_client::types::DatasetEnsureBody {
 )]
 #[serde(tag = "type", rename_all = "snake_case")]
 pub enum ServiceType {
-    Nexus { internal_ip: Ipv6Addr, external_ip: IpAddr },
-    InternalDns { server_address: SocketAddrV6, dns_address: SocketAddrV6 },
+    Nexus {
+        internal_ip: Ipv6Addr,
+        external_ip: IpAddr,
+    },
+    InternalDns {
+        server_address: SocketAddrV6,
+        dns_address: SocketAddrV6,
+    },
     Oximeter,
     ManagementGatewayService,
     Wicketd,
-    Dendrite { asic: DendriteAsic },
-    Tfport { pkt_source: String },
+    Dendrite {
+        asic: DendriteAsic,
+    },
+    Tfport {
+        pkt_source: String,
+    },
     CruciblePantry,
-    Ntp { servers: Vec<String>, boundary: bool },
-    DnsClient { servers: Vec<String>, domain: Option<String> },
```

Comment:
This is the only non-formatting change -- the arguments to launch the `DnsClient` are passed through the NTP server